public/css/main.css

```diff
@@ -35,51 +35,45 @@
 .dcricon-decred:before {
   content: "\e900";
 }
-
 /* layout */
 .container.main {
   padding: 20px;
 }
 .row .col-md-4 table {
   margin: auto;
-  margin-right: 0px;
+  margin-right: 0;
 }
-
 /*fonts*/
 @font-face {
   font-family: 'source-sans-pro-v9-latin-regular';
-  src:  url('/fonts/source-sans-pro-v9-latin-regular.eot');
-  src:  url('/fonts/source-sans-pro-v9-latin-regular.eot#iefix') format('embedded-opentype'),
-  url('/fonts/source-sans-pro-v9-latin-regular.woff') format('woff'),
-  url('/fonts/source-sans-pro-v9-latin-regular.svg') format('svg');
+  src: url('/fonts/source-sans-pro-v9-latin-regular.eot');
+  src: url('/fonts/source-sans-pro-v9-latin-regular.eot#iefix') format('embedded-opentype'),
+    url('/fonts/source-sans-pro-v9-latin-regular.woff') format('woff'),
+    url('/fonts/source-sans-pro-v9-latin-regular.svg') format('svg');
   font-weight: normal;
   font-style: normal;
 }
 @font-face {
   font-family: 'inconsolata-v15-latin-regular';
-  src:  url('/fonts/inconsolata-v15-latin-regular.eot');
-  src:  url('/fonts/inconsolata-v15-latin-regular.eot#iefix') format('embedded-opentype'),
-  url('/fonts/inconsolata-v15-latin-regular.woff') format('woff'),
-  url('/fonts/inconsolata-v15-latin-regular.svg') format('svg');
+  src: url('/fonts/inconsolata-v15-latin-regular.eot');
+  src: url('/fonts/inconsolata-v15-latin-regular.eot#iefix') format('embedded-opentype'), url('/fonts/inconsolata-v15-latin-regular.woff') format('woff'), url('/fonts/inconsolata-v15-latin-regular.svg') format('svg');
   font-weight: normal;
   font-style: normal;
 }
-
 /*elements*/
 html {
   color: #052f4b;
   -moz-osx-font-smoothing: grayscale;
   -webkit-font-smoothing: antialiased !important;
   -moz-font-smoothing: antialiased !important;
   text-rendering: optimizelegibility !important;
-  padding-top: 35px;
```

Comment:
Please describe all the UI changes.

Reply:
html had padding top, body had padding bottom. I took off both and gave body a padding top and bottom. All page visible content should be in body, that's what makes sense to pad.

Reply:
Most other CSS changes were eliminating unnecessary nesting of selectors

Reply:
In nexthome.css you can go to town, since the css is majorly screwed up and you've effectively taken over ownership of that. But for main.css which several current contributors have worked on I think it's valuable to preserve the git history. 

Although I think the stylistic changes you're making to main.css are mostly fine, I don't see them as necessary  and that us all striving for good git hygiene is much more beneficial.

Reply:
Noted. Will undo changes to main.css

Reply:
Thanks, @gozart1.  I strongly agree with the points you made.